documentation/asciidoc/topics/proc_exposing_loadbalancer.adoc

```diff
@@ -0,0 +1,47 @@
+[id='exposing_loadbalancer-{context}']
+= Exposing {brandname} Through Load Balancers
+Use a load balancer service to make {brandname} clusters available to clients running outside {k8s}.
+
+[NOTE]
+====
+To access {brandname} with unencrypted Hot Rod client connections you must use
+a load balancer service.
+====
+
+.Procedure
+
+. Include `spec.expose` in your **Infinispan** CR.
+. Specify `LoadBalancer` as the service type with `spec.expose.type`.
++
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  expose:
+    type: LoadBalancer <1>
+----
++
+<1> Exposes {brandname} on the network through a load balancer service.
++
+By default the exposed port is `11222`. You can optionally change the port with
```

Comment:
I'm not sure, we need to discuss it with @rigazilla but looks we can't change an exposition port, it always 11222 for LoadBalancer and just possible to change NodePort for this. Maybe it depends of the LoadBalancer provider, for example AWS
```
16:57 $ oc get service | grep -external
example-infinispan-external   LoadBalancer   172.30.127.24   aedc4e6a87e9346e7928a27759f68e60-2087709112.eu-central-1.elb.amazonaws.com   11222:30000/TCP   92m
```

Reply:
I did a test with QuickLab and LoadBalancer, but since QuickLab didn't provide external additional ip address, I've tested using a node address, so my results aren't really reliable.

Reply:
@rigazilla , yes, looks like if LoadBalancer function is not available at the platform, the service similar to NodePort will created
Let's add @Crumby for review

Reply:
@dmvolod I asked @Crumby who said:

`spec.expose.nodePort` applies to both actually but it's less relevant for the LoadBalancer. If I understand it correctly it's basically super structure on top of NodePort.... LoadBalancer creates a route provided by platform that redirects request to node port redirecting request to service...

It's basically loadBalancerUrl:11222 -> 'nodeHostName:nodePort' -> 'service:11222'

Reply:
Yes, @oraNod sure. It's a good explanation. In our case NodePort for LoadBalancer is less important from user experience point of view.